nav2_bringup/bringup/rviz/nav2_namespaced_view.rviz

```diff
@@ -164,24 +164,19 @@ Visualization Manager:
       Use Timestamp: false
       Value: true
     - Alpha: 1
-      Arrow Length: 0.019999999552965164
-      Axes Length: 0.30000001192092896
-      Axes Radius: 0.009999999776482582
-      Class: rviz_default_plugins/PoseArray
+      Class: nav2_rviz_plugins/ParticleCloudDisplay
       Color: 0; 180; 0
       Enabled: true
-      Head Length: 0.07000000029802322
-      Head Radius: 0.029999999329447746
+      Max Arrow Length: 0.10000000149011612
+      Min Arrow Length: 0.019999999552965164
       Name: Amcl Particle Swarm
-      Shaft Length: 0.23000000417232513
-      Shaft Radius: 0.009999999776482582
       Shape: Arrow (Flat)
       Topic:
         Depth: 5
         Durability Policy: Volatile
         History Policy: Keep Last
         Reliability Policy: Best Effort
-        Value: <robot_namespace>/particlecloud
+        Value: <robot_namespace>/particle_cloud
```

Comment:
I think this is the only change you actually need to make, can you revert the other rviz parameters? Or is there some reason the other updates are required?

Reply:
This plugin scales the length and radius of the arrows according to the weight of the particle. What that means is that each particle is represented with an arrow of possibly a different length based on its weight. Those parameters that I have removed defined the length and radius of all the arrows (like how it is with PoseArray display).

I have instead defined two parameters, minimum and maximum arrow length. The arrows are scaled to fit this range based on their weights.

Reply:
Why did that involve resizing the window, disabling the TF tree, state, update topic, etc. I'm not concerned about the params under `nav2_rviz_plugins/ParticleCloudDisplay`, but the other things you modified in the rviz configs if you look at the diff